build.sbt

```diff
@@ -47,43 +47,30 @@ val exampleCases: List[(java.io.File, String, Boolean, List[String])] = List(
   (sampleResource("pathological-parameters.yaml"), "pathological", false, List.empty)
 )
 
-val exampleJavaArgs: List[List[String]] = exampleCases
-  .foldLeft(List[List[String]](List("java")))({
-    case (acc, (path, prefix, tracing, extra)) =>
-      acc ++ (for {
-        kind <- List("client", "server")
-        frameworkPair <- List(
-          ("dropwizard", "dropwizard"),
-        )
-        (frameworkName, frameworkPackage) = frameworkPair
-        tracingFlag                       = if (tracing) Option("--tracing") else Option.empty[String]
-      } yield
-        (
-          List(s"--${kind}") ++
-            List("--specPath", path.toString()) ++
-            List("--outputPath", s"modules/sample-${frameworkPackage}/src/main/java/generated") ++
-            List("--packageName", s"${prefix}.${kind}.${frameworkPackage}") ++
-            List("--framework", frameworkName)
-        ) ++ tracingFlag ++ extra)
-  })
+val exampleFrameworkSuites = Map(
```

Comment:
Why use a map? Why not just
```scala
object FrameworkSuites {
  val java = List(...)
  val scala = List(...)
}
```
?

Reply:
Cuz then I'm passing a `List[(String, String, List[String])]` to `exampleArgs()`, which is fine, but then it wants a string to represent the language to pass as an argument, and... it's gone now.

Reply:
Ah, that makes sense.